src/neo/SmartContract/ApplicationEngine.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2015-2021 The Neo Project.
+// Copyright (C) 2015-2022 The Neo Project.
```

Comment:
I realize this is nit-picking, but can we get someone to do a pass thru all the code and update the copyrights instead of updating them one at a time in unrelated PRs?